examples/apps/djapp/3_add_crds/virtual-node-definition.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: virtualnodes.appmesh.k8s.aws
+spec:
+  group: appmesh.k8s.aws
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+    - name: v1alpha1
+      served: true
+      storage: false
+  version: v1beta1
+  scope: Namespaced
+  names:
+    plural: virtualnodes
+    singular: virtualnode
+    kind: VirtualNode
+    categories:
+      - all
+      - appmesh
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      required:
+        - spec
+      properties:
+        spec:
+          required:
+            - meshName
+          properties:
+            meshName:
+              type: string
+            listeners:
+              type: array
+              items:
+                type: object
+                properties:
+                  portMapping:
+                    properties:
+                      port:
+                        type: integer
+                      protocol:
+                        type: string
+                        enum:
+                          - tcp
+                          - http
+                          - grpc
+                          - http2
+                          - https
```

Comment:
Hmm, these are not all currently valid (https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_PortMapping.html). 

Brings up a general question of how we intend to modify this demo as new features are added?